syntax = "proto3";

message LocationItem {
  int32 person_id = 1;
  string latitude = 2;
  string longitude = 3;
  string creation_time = 4;
}

service LocationService {
  rpc Ingest(LocationItem) returns (LocationItem);
}